haystack/utils/deepsetcloud.py

```diff
@@ -726,7 +747,7 @@ def get_labels(self, evaluation_set: Optional[str], workspace: Optional[str] = N
         """
         try:
             evaluation_sets_response = next(
-                self._get_evaluation_set(evaluation_set=evaluation_set, workspace=workspace)
+                self._get_evaluation_sets(evaluation_set_filter=evaluation_set, workspace=workspace)
```

Comment:
Could you please briefly explain the renaming? I understand what an evaluation_set set is but what is an evaluation_set_filter? 🙂  🤔

Reply:
Setting this param filters the evaluation sets by name. The api does that by using a contains-like comparison. Maybe renaming it to `name_filter` better expresses this.

Reply:
Thanks for the pointer. I had a look at the DC code and the string that is passed there could even be a regex, for example `"[0-9]?"`. I think the name `evaluation_set_filter` is okay as `evaluation_set ` is in Haystack always referring to the **name** of the evaluation set. Can be a bit confusing though. 
Could you please update the `:param` doc string for `evaluation_set` of the `get_labels` method so that it explains that one can pass not only the exact name but also a substring (or even a regex if this will be supported also in future).

Reply:
I would prefer `evaluation_set_filter` over `name_filter`.